test/php/library/Director/Objects/IcingaServiceTest.php

```diff
@@ -12,6 +12,7 @@ class IcingaServiceTest extends BaseTestCase
     protected $testHostName = '___TEST___host';
 
     protected $testServiceName = '___TEST___service';
+    protected $createdServices = [];
```

Comment:
We also hate it, but we force ourselves to stay compatible with PHP 5.3 as there are still "Enterprise" Linuxes shipping with that one. Please use array() instead of [] here (and in other places).

Thanks,
Thomas